lib/Backend/LinearScan.cpp

```diff
@@ -3338,12 +3338,6 @@ LinearScan::KillImplicitRegs(IR::Instr *instr)
         this->RecordLoopUse(nullptr, LowererMDArch::GetRegIMulHighDestLower());
         return;
     }
-
-    if (instr->m_opcode == Js::OpCode::Yield)
-    {
-        this->bailIn.SpillRegsForBailIn();
-        return;
```

Comment:
The Yield opcode is removed much earlier in the Jit so this condition was always false. Note "fixing" the condition so it could be true in the intended circumstances introduced additional segfaults and no benefits hence removed.